api/database/migrations/2025_01_30_180016_rename_user_gov_columns.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->renameColumn('is_gov_employee', 'computed_is_gov_employee');
+            $table->renameColumn('gov_employee_type', 'computed_gov_employee_type');
+            $table->renameColumn('current_classification', 'computed_classification');
+            $table->renameColumn('department', 'computed_department');
+            $table->text('computed_gov_position_type')->nullable();
```

Comment:
On a tangent given sometimes we use `->text()` and sometimes `->string()`
Apparently it doesn't matter too much and text is usually recommended as a default choice 🤔 https://stackoverflow.com/a/4849030

Reply:
Or maybe not, it might be heated debate 😅 

![image](https://github.com/user-attachments/assets/8415de80-9459-4ea4-b3e4-7fe24de9accf)

Reply:
Yeah, I lean towards `text` as the default. I think `string` is better when you know the max length is only a few characters